crates/hyperdrive-math/src/short/open.rs

```diff
@@ -548,6 +634,63 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn fuzz_calculate_spot_price_after_short_defaults() -> Result<()> {
```

Comment:
after_short_deltas?

Reply:
no it's a specific test for the default behavior. But I'll move the word to make it more clear.